docs/self-managed/backup-restore/zeebe-backup-and-restore.md

```diff
@@ -0,0 +1,170 @@
+
+- 200 OK, with response body
+- 500 Server error
+
+##### Response Body
+
+```
+{
+  id: <backupId>
+}
+```
+
+The response is sent after Zeebe has started taking the backup of all partitions, but before the backup is completed. The backup is taken asynchronously and can take a long time depending on the size. To monitor the backup, use the get backup status api.
+
+### Monitor backup
+
+##### Request
+
+```
+GET http://{zeebe-gateway}:9600/actuator/backups/{BackupId}
+```
+
+##### Response
+
+- 200 OK, with response body
+- 500 Server error
+
+##### Response Body
+
+```
+{
+    "backupId": 6,
+    "status": "COMPLETED", // COMPLETED | IN_PROGRESS | FAILED | DOES_NOT_EXIST
+    "failureReason": null,
+    "partitions": [
+        {
+            "partitionId": 1,
+            "status": "COMPLETED",
+            "description": {
+                "snapshotId": "16-10-7-8",
+                "checkpointPosition": 9,
+                "brokerId": 1,
+                "brokerVersion": "8.1.0"
+            },
+            "failureReason": null,
+            "createdAt": "2022-09-26T14:57:17.914044812Z",
+            "lastUpdatedAt": "2022-09-26T14:57:19.107670926Z"
+        },
+        {
+            "partitionId": 2,
+            "status": "COMPLETED",
+            "description": {
+                "snapshotId": "18-12-7-8",
+                "checkpointPosition": 9,
+                "brokerId": 0,
+                "brokerVersion": "8.1.0"
+            },
+            "failureReason": null,
+            "createdAt": "2022-09-26T14:57:17.914367447Z",
+            "lastUpdatedAt": "2022-09-26T14:57:18.887925972Z"
+        }
+    ]
+}
+
+```
+
+`status` gives the overall status of the backup. It is
+
+- `COMPLETED` if all partitions have completed the backup
+- `FAILED` if at least one partition have failed. In this case `failureReason` contains a string describing the reason for failure.
+- `DOES_NOT_EXIST` if at least one partition's backup does not exist
+- `IN_PROGRESS` if at least one partition's backup is in progress
+
+## Restore
+
+A new Zeebe cluster can be created from a specific backup. We provide a standalone app which must be run on each node where a Zeebe broker will be running. This is a Spring Boot application similar to the broker and can run using the binary provided as part of the distribution. The app can be configured the same way a broker is configured - via environment variables or using the configuration file located in `config/application.yaml`.
+
+To restore a zeebe cluster, run the following in each node where the broker will be running.
+
+```
+tar -xzf zeebe-distribution-X.Y.Z.tar.gz -C zeebe/
+./bin/restore <backupId>
+```
+
+If restore was successful, the app exits with a log message `Successfully restored broker from backup`.
+
+Restore fails if:
+
+- There is no valid backup with the given backupId
+- Backup store is not configured correctly
+- The configured data directory is not empty
+- Any other unexpected errors
```

Comment:
```suggestion
- There is no valid backup with the given `backupId`.
- Backup store is not configured correctly.
- The configured data directory is not empty.
- Any other unexpected errors.
```

Reply:
This is a bit of a nitpick, but since they read like sentences, I suggest periods.